src/app/models/equipment.ts

```diff
@@ -0,0 +1,7 @@
+import { AttributeModifier } from './attribute-modifier';
+
+export class Equipment {
```

Comment:
why do you need this for the navbar?

Reply:
Navbar doesnt need this directly, mocked heroes hall's dummy heroes required in its model.